dao/accumulo.rya/src/main/java/org/apache/rya/accumulo/instance/AccumuloRyaInstanceDetailsRepository.java

```diff
@@ -89,12 +89,17 @@ public AccumuloRyaInstanceDetailsRepository(final Connector connector, final Str
 
     @Override
     public boolean isInitialized() throws RyaDetailsRepositoryException {
+        Scanner scanner = null;
         try {
```

Comment:
Actually, never mind. The Accumulo Scanner interface doesn't implement AutoCloseable.